spack.yaml

```diff
@@ -1,16 +1,21 @@
+# Copyright 2024 ACCESS-NRI and contributors. See the top-level COPYRIGHT file for details.
+# SPDX-License-Identifier: Apache-2.0
+#
 # This is a Spack Environment file.
 #
 # It describes a set of packages to be installed, along with
 # configuration settings.
 spack:
   # add package specs to the `specs` list
   specs:
-  - access-om2@git.2023.11.23
+  - access-om2@git.2024.03.0
   packages:
     cice5:
       require: '@git.2023.10.19'
     mom5:
-      require: '@git.2023.11.09'
+      require: 
+        - '@git.2023.11.09' 
+        - 'type=ACCESS-OM-BGC'
```

Comment:
Will need to go through the `packages.yaml` syntax to see this syntax - will have to update the CI if this is a possibility (specifically, the `yq` that gets the version - thats why https://github.com/ACCESS-NRI/ACCESS-OM2-BGC/actions/runs/8367053860/job/22908627210?pr=2#step:5:28 is failing)

Reply:
Won't be relevant any more, but would still be good to find out - didn't expect that syntax as a possibility!